azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -804,6 +771,77 @@ private void getToken(String userToProxy) throws InterruptedException,
 
   }
 
+  /**
+   * Prepare token file.
+   *  Writes credentials to a token file and sets permissions such that only the user being proxied has read access
```

Comment:
Technically the azkaban user has read access too. 

If you would prefer avoiding duplicating comments, may be point to the comment of the other method?